data/sql/updates/pending_db_world/rev_1592650543169399600.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1592650543169399600');
+
+UPDATE `smart_scripts` SET `link` = 0 WHERE `entryorguid` IN (16398,8338,178908,32347,24077) AND `id` = 0 OR `entryorguid` = 30895 AND `id` = 1;
```

Comment:
this is wrong, you need to put parenthesis to avoid unexpected results:

WHERE (cond1 **AND** cond2) **OR** (cond3 **AND** cond4)

```sql
UPDATE `smart_scripts` SET `link` = 0
WHERE (`entryorguid` IN (16398,8338,178908,32347,24077) AND `id` = 0);
OR (`entryorguid` = 30895 AND `id` = 1);
```

Reply:
Maybe, but that worked too :shrug: 

Will update later

Reply:
@Kitzunu that worked but you don't know if it caused unexpected side-effect (i.e. updating other scripts that weren't meant to be changed)

I suggest to install a clean world DB and try my query